app/models/manageiq/providers/openstack/network_manager/refresh_parser.rb

```diff
@@ -37,13 +37,11 @@ def ems_inv_to_hashes
       get_security_groups
       get_networks
       get_network_routers
-      get_floating_ips
       get_network_ports
+      get_floating_ips
```

Comment:
right now if they will refactor, the tests will blow up :-)

but yeah, at least comment in the start? I think the ultimate plan is to convert this to some sort of dependency graph, where you could do parts in parallel